quinn-proto/src/connection/streams.rs

```diff
@@ -423,6 +424,25 @@ impl Streams {
         Ok(())
     }
 
+    pub fn set_priority(&mut self, id: StreamId, priority: i32) -> Result<(), UnknownStream> {
+        let stream = match self.send.get_mut(&id) {
+            Some(ss) => ss,
+            None => return Err(UnknownStream { _private: () }),
+        };
+
+        stream.priority = priority;
+        Ok(())
+    }
+
+    pub fn get_priority(&mut self, id: StreamId) -> Result<i32, UnknownStream> {
```

Comment:
Should we adhere to the API guidelines here and call these just `priority()` rather than using the `get_` prefix?

Reply:
Good catch; `priority` is definitely more idiomatic.

Reply:
Otherwise this looks good to me, feel free to merge when this is fixed.

Reply:
Fixed.